_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,113 @@
+
+<!-- Video of Final -->
+<iframe
+  class="center video"
+  src="https://www.youtube.com/embed/gOeP96OngE0"
+  frameborder="0"
+  width="100%"
+  allowfullscreen
+  >
+</iframe>
+
+We saw some brilliant robot designs this year, great work by all teams. We hope you enjoyed the competition as much as we did!
+
+If you didn't take part, or you want to enter again next year, the [sign up page](/schools/how_to_enter) for next year's competition will be up later in the year. Get a team together and start talking to your teachers now!
+
+Results
+-------
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| <span title="The rookie team who achieve the highest place in the league">Rookie Award</span>          | [Eltham College](https://www.elthamcollege.london/)
+| <span title="The team that displays the most extraordinary ingenuity in the design of their robot">Committee Award</span>       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| <span title="The team that presents their robot and themselves in what is judged to be the most outstanding way">Robot and Team Image</span>  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| <span title="The team that is judged to have the best online presence using the hashtag #srobo19">Online Presence</span>       | [Collyer's](https://www.collyers.ac.uk/) ([Twitter](https://twitter.com/CollyersRobots))
+
+We always like to see new teams joining the competition, and Eltham College were still able to pull their weight against our seasoned competitors. Their robot picked up a stack of tokens and used tank tracks as a method of climbing the volcano.
+
+The Committee Award is especially important to us. As engineers, we are looking for the most simple and elegant solution to every challenge, and Team SWI from South Wilts Grammar School managed to do exactly that. Their robot was designed to take a single token into the highest scoring zone, the Caldera, by using [large wheels to climb the volcano](https://twitter.com/studentrobotics/status/1114534154283175936). Over the course of the competition weekend, they made notes of other teams' strategies and were constantly evolving their own to stay competitive. They also quickly and cleverly changed their approach to overcome technical difficulties midway through the competition.
```

Comment:
The capitalisation of "Caldera" used here to refer to the particular scoring zone is different to that used elsewhere. I don't mind whether we always quote it or always capitalise it, however I would prefer that we are consistent.

Note that previously, the capitalised form has also been used to refer to the game as a whole.

Reply:
https://github.com/srobo/website/pull/148/commits/b81f9fcd5f5951f9d935a9140452b81606ff381a